src/main/java/FileContent.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Class for reading and writing content from File
+ */
+public class FileContent implements Content {
+
+    private final File file;
+    private final String charsetName;
+
+    /**
+     * Constructor for initializing FileContent object
```

Comment:
@ArthurRomasyuk 
`Ctor.` just enough. It is constructor, hence known it initializes its class objects. No need to provide anything else, except if there is something you'd definitely want client code to be aware about. Like opens the file/connection etc, which is not our case